app/views/static/references.html.erb

```diff
@@ -9,30 +9,81 @@
+        <p class='static-page__text static-page__text--sinai'>G&eacute;hin 2017 = G&eacute;hin, Paul. <i>Les manuscrits syriaques de parchemin du Sina&iuml; et leurs</i>membra disjecta (CSCO 665, Subsidia 136; Leuven, 2017). [<%= link_to 'WorldCat', 'https://www.worldcat.org/title/manuscrits-syriaques-de-parchemin-du-sinai-et-leurs-membra-disjecta/oclc/1000455293%26referer%3Dbrief_results&amp;sa=D&amp;source=editors&amp;ust=1683922505980598&amp;usg=AOvVaw2ZbqofVswP2V57Y1V5Z1Z_"', class: 'si-link' %>]</p>
 
-        <p class='static-page__text static-page__text--sinai'>Philothée 2008 = Philothée du Sinaï, <i>Nouveaux manuscrits syriaques du Sinaï</i> (Athens, 2008). [<%= link_to 'WorldCat', "https://www.worldcat.org/title/nouveaux-manuscrits-syriaques-du-sinai/oclc/476756980?referer=br&ht=edition", class: 'si-link' %>]</p>
+        <p class='static-page__text static-page__text--sinai'>Gibson = Gibson, Margaret Dunlop. <i>Catalogue of the Arabic Manuscripts in the Convent of S. Catharine on Mount Sinai</i>. Cambridge University Press, 1894.[<%= link_to 'Internet Archive', 'https://archive.org/details/catalogueofarabi00sainrich&amp;sa=D&amp;source=editors&amp;ust=1683922505981437&amp;usg=AOvVaw0O-pBD3AlmXKj057ALKt5h"', class: 'si-link' %>]</p>
+
+        <p class='static-page__text static-page__text--sinai'>Greek New Finds = Nikolopoulos, P. G. &Tau;&alpha; &nu;&#941;&alpha; &epsilon;&upsilon;&rho;&#942;&mu;&alpha;&tau;&alpha; &tau;&omicron;&upsilon; &Sigma;&iota;&nu;&#940; (Athens, 1998). English translation: A. Hatzopoulos, The New Finds of Sinai (Athens, 1999). [<%= link_to 'WorldCat', 'https://www.worldcat.org/title/new-finds-of-sinai/oclc/913489510%26referer%3Dbrief_results&amp;sa=D&amp;source=editors&amp;ust=1683922505981979&amp;usg=AOvVaw2gGrj5nM76A_YNI2kLJPtF"', class: 'si-link' %>]</p>
+
+        <p class='static-page__text static-page__text--sinai'>Kamil = K&#257;mil, Mur&#257;d. <i>Catalogue of All Manuscripts in the Monastery of St. Catherine on Mount Sinai</i>. Harrassowitz, 1970. [<%= link_to 'WorldCat', 'https://www.worldcat.org/title/catalogue-of-all-manuscripts-in-the-monastery-of-st-catharine-on-mount-sinai/oclc/859613867?referer%3Dbr%26ht%3Dedition&amp;sa=D&amp;source=editors&amp;ust=1683922505982771&amp;usg=AOvVaw2eubcGMhjKIcgPk9rRia8C"', class: 'si-link' %>] [<%= link_to 'WorldCat', 'https://www.worldcat.org/oclc/42432167&amp;sa=D&amp;source=editors&amp;ust=1683922505983084&amp;usg=AOvVaw38nfNYq_H1cjtKjVMq3sbv"', class: 'si-link' %>]</p>
```

Comment:
The second link here, `[<%= link_to 'WorldCat', 'https://www.worldcat.org/oclc/42432167&amp;sa=D&amp;source=editors&amp;ust=1683922505983084&amp;usg=AOvVaw38nfNYq_H1cjtKjVMq3sbv"', class: 'si-link' %>]` can be removed. This was my fault as it slipped through my proofreading of the Google doc, apologies

Reply:
The URL for the first link should be: https://www.worldcat.org/title/catalogue-of-all-manuscripts-in-the-monastery-of-st-catharine-on-mount-sinai/oclc/859613867